apps/application-system/api/src/app/modules/application/application.controller.ts

```diff
@@ -116,6 +118,7 @@ export class ApplicationController {
   ) {}
 
   @Scopes(ApplicationScope.read)
+  @UseGuards(DelegationGuard)
```

Comment:
The `DelegationGuard` is applied to all methods expect for the `delete`, shouldn't it be there as well? Could we just add it in the `UseGuards`  on the controller level?